web_widget_text_markdown/README.rst

```diff
@@ -0,0 +1,105 @@
+========================
+Web Widget Text Markdown
+========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/12.0/web_widget_text_markdown
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_widget_text_markdown
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds a new widget for text field in form view on Odoo:
+
+- In readonly mode, it uses text contents to parse and render them to html markdown syntax.
+- In write mode, use [bootstrap-markdown][1]
+
+[1]: `bootstrap-markdown <https://github.com/toopay/bootstrap-markdown>`_
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Your XML form view definition should contain::
+
+    ...
+    <field name="field_name" widget="bootstrap_markdown"/>
+    ...
+
+Known issues / Roadmap
+======================
+
+* Can't create attachments on virtual records
+
+* Enable dropzone support (Disabled for support IE11)
```

Comment:
Perhaps now is the time to enable dropzone support? Since the latest dropzone version seems to support IE11 https://www.dropzonejs.com/#compatibility

Reply:
Not yet... http://www.odoo.com/documentation/11.0/setup/deploy.html#supported-browsers

Reply:
Yes, sorry... :P dropzone is used to allow attach images ;)